pyproject.toml

```diff
@@ -1,7 +1,79 @@
+# See https://packaging.python.org/en/latest/specifications/declaring-project-metadata
+
+[tool.poetry]
+name = "dsp-tools"
+version = "1.22.2"
+description = "DSP-TOOLS is a Python package with a command line interface that helps you interact with a DaSCH service platform (DSP) server."
+authors = ["DaSCH - Swiss National Data and Service Center for the Humanities <info@dasch.swiss>"]
+readme = "docs/index.md"
+license = "GPL-3.0-only"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
```

Comment:
Should this be switched to Apache? I think in most other repos we've done so

Reply:
I'm open for it, but I don't understand the legal implications, and I don't know who has the competence to decide it.